modular_skyrat/modules/food_replicator/replicator_designs/replicator_food.dm

```diff
@@ -0,0 +1,111 @@
+/datum/design/ration
+	name = "foreign colonization ration"
+	id = "slavic_mre"
+	build_type = BIOGENERATOR
+	materials = list(/datum/material/biomass = 550)
+	build_path = /obj/item/storage/box/colonial_rations
+	category = list(RND_CATEGORY_INITIAL, RND_CATEGORY_NRI_FOOD)
+
+/datum/design/pljeskavica
+	name = "foreign colonization ration, main course"
+	id = "slavic_burger"
+	build_type = BIOGENERATOR
+	materials = list(/datum/material/biomass = 200)
+	build_path = /obj/item/food/colonial_course/pljeskavica
+	category = list(RND_CATEGORY_INITIAL, RND_CATEGORY_NRI_FOOD)
+
+/datum/design/nachos
+	name = "foreign colonization ration, side dish"
+	id = "mexican_chips"
+	build_type = BIOGENERATOR
+	materials = list(/datum/material/biomass = 150)
+	build_path = /obj/item/food/colonial_course/nachos
+	category = list(RND_CATEGORY_INITIAL, RND_CATEGORY_NRI_FOOD)
+
+/datum/design/blins
+	name = "foreign colonization ration, dessert"
+	id = "slavic_crepes"
+	build_type = BIOGENERATOR
+	materials = list(/datum/material/biomass = 100)
+	build_path = /obj/item/food/colonial_course/blins
+	category = list(RND_CATEGORY_INITIAL, RND_CATEGORY_NRI_FOOD)
+
+/datum/design/spork
+	name = "foreign colonization ration, utensils"
+	id = "slavic_utens"
+	build_type = BIOGENERATOR
+	materials = list(/datum/material/biomass = 75)
+	build_path = /obj/item/storage/box/utensil
+	category = list(RND_CATEGORY_INITIAL, RND_CATEGORY_NRI_FOOD)
+
+/datum/design/fasani
+	name = "foreign colonization ration, bubblegum pack"
+	id = "slavic_gum"
+	build_type = BIOGENERATOR
+	materials = list(/datum/material/biomass = 100)
+	build_path = /obj/item/storage/box/gum/colonial
+	category = list(RND_CATEGORY_INITIAL, RND_CATEGORY_NRI_FOOD)
+
+/datum/design/cup
+	name = "paper cup, stylised, empty"
```

Comment:
This is an incredibly long name, I feel like, when it could just be "stylised paper cup", no?

Reply:
Plan was to mimic the naming scheme of various MRE products' manufactured in my country and the adjacent ones, with their properties listed after commas. On a second thought that is pretty weird; will replace that.